recipe/meta.yaml

```diff
@@ -12,11 +10,15 @@ package:
 
 source:
   fn: conda-{{ version }}.tar.gz
-  url: https://github.com/conda/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/conda/conda/releases/download/{{ version }}/conda-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    # Backport https://github.com/conda/conda/pull/14117 for changes in the
+    # Python 3.13+ logging module internals
+    - patches/0001-GH-14117-logging-lock-internal-changes-py313.patch
```

Comment:
Have merged in Bradley's PR ( https://github.com/conda-forge/conda-feedstock/pull/258 ), which the patch from upstream PR ( https://github.com/conda/conda/pull/14117 )

Reply:
Thanks Jonathan! 🙏

Glad to see your fix landed 🙂

Yeah we can update the patch. Would you like to send a PR to update it?